services/api/service.go

```diff
@@ -945,15 +1045,44 @@ func (api *RelayAPI) handleSubmitNewBlock(w http.ResponseWriter, req *http.Reque
 		}
 	}
 
-	builderIsHighPrio, builderIsBlacklisted, err := api.redis.GetBlockBuilderStatus(payload.Message.BuilderPubkey.String())
+	builderStatusCode, err := api.redis.GetBlockBuilderStatus(payload.Message.BuilderPubkey.String())
 	log = log.WithFields(logrus.Fields{
-		"builderIsHighPrio":    builderIsHighPrio,
-		"builderIsBlacklisted": builderIsBlacklisted,
+		"builderStatus":    builderStatusCode,
```

Comment:
Consider normalising the use of `builderStatus` and `builderStatusCode` throughout.

Reply:
I would just call it "Status" everywhere (remove the word "Code").

Reply:
ya that is what i was just doing :-) code is for sure unnecessary. it was only there from when we still had the redis type `BlockBuilderStatus`. should be good now, but lmk if you see any stragglers!